### DISCUSSION TITLE
Get X/Y of touch

### DISCUSSION THREAD

**Discussion by BinaryAssault** [dotnet/maui#10716] [Q&A]
Are there any controls that allow you to get the X/Y location when a user touches the control? I can't find any.

**Comment by KSemenenko** [comment]
I don’t think it’s supported right now

**Comment by xtuzy** [comment]
Use like https://github.com/vapolia/MauiGestures add a Gesture to get position or use GraphicsView

**Reply by BinaryAssault** [reply]
This looks promising, thank you for this
